strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/AffectedRunnerProperties.java

```diff
@@ -0,0 +1,61 @@
+package org.arquillian.smart.testing.strategies.affected;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class AffectedRunnerProperties {
+
+    public static final String SMART_TESTING_AFFECTED_EXCLUSIONS = "smart.testing.affected.exclusions";
+    public static final String SMART_TESTING_AFFECTED_INCLUSIONS = "smart.testing.affected.inclusions";
+    public static final String SMART_TESTING_AFFECTED_EXCLUSIONS_FILE = "smart.testing.affected.exclusions.file";
+    public static final String SMART_TESTING_AFFECTED_INCLUSIONS_FILE = "smart.testing.affected.inclusions.file";
+
+    public static final String getSmartTestingAffectedExclusions() {
+        String exclusions = System.getProperty(SMART_TESTING_AFFECTED_EXCLUSIONS);
+        String exclusionsFile = readFile(System.getProperty(SMART_TESTING_AFFECTED_EXCLUSIONS_FILE));
+
+        StringJoiner joiner = new StringJoiner(", ");
+        if (exclusions != null) {
+            joiner.add(exclusions);
+        }
+
+        if (exclusionsFile != null) {
+            joiner.add(exclusionsFile);
+        }
+
+        return joiner.toString().trim();
+    }
+
+    public static final String getSmartTestingAffectedInclusions() {
+        String inclusions = System.getProperty(SMART_TESTING_AFFECTED_INCLUSIONS);
+        String inclusionsFile = readFile(System.getProperty(SMART_TESTING_AFFECTED_INCLUSIONS_FILE));
+
+        StringJoiner joiner = new StringJoiner(", ");
+        if (inclusions != null) {
+            joiner.add(inclusions);
+        }
+
+        if (inclusionsFile != null) {
+            joiner.add(inclusionsFile);
+        }
+
+        return joiner.toString().trim();
+    }
+
+    private static String readFile(String location) {
+        if (location == null) {
+            return null;
+        }
+
+        try (Stream<String> lines = Files.lines(Paths.get(location))) {
+            return lines.collect(Collectors.joining(", ")).trim();
```

Comment:
The string `", "` could be as a constant and reuse it also here https://github.com/arquillian/smart-testing/pull/99/files#diff-83c6b0e6ef034dfbc8970a4e1ed134f8R32